cmd/chartmuseum/main.go

```diff
@@ -75,49 +75,50 @@ func cliHandler(c *cli.Context) {
-		EnforceSemver2:         conf.GetBool("enforce-semver2"),
-		CacheInterval:          conf.GetDuration("cacheinterval"),
-		Host:                   conf.GetString("listen.host"),
-		PerChartLimit:          conf.GetInt("per-chart-limit"),
-		WebTemplatePath:        conf.GetString("web-template-path"),
-		ArtifactHubRepoID:      conf.GetStringMapString("artifact-hub-repo-id"),
+		Version:                 Version,
+		StorageBackend:          backend,
+		ExternalCacheStore:      store,
+		Logger:                  logger,
+		TimestampTolerance:      conf.GetDuration("storage.timestamptolerance"),
+		ChartURL:                conf.GetString("charturl"),
+		TlsCert:                 conf.GetString("tls.cert"),
+		TlsKey:                  conf.GetString("tls.key"),
+		TlsCACert:               conf.GetString("tls.cacert"),
+		Username:                conf.GetString("basicauth.user"),
+		Password:                conf.GetString("basicauth.pass"),
+		ChartPostFormFieldName:  conf.GetString("chartpostformfieldname"),
+		ProvPostFormFieldName:   conf.GetString("provpostformfieldname"),
+		ContextPath:             conf.GetString("contextpath"),
+		LogHealth:               conf.GetBool("loghealth"),
+		LogLatencyInteger:       conf.GetBool("loglatencyinteger"),
+		EnableAPI:               !conf.GetBool("disableapi"),
+		DisableDelete:           conf.GetBool("disabledelete"),
+		UseStatefiles:           !conf.GetBool("disablestatefiles"),
+		AllowOverwrite:          conf.GetBool("allowoverwrite"),
+		AllowForceOverwrite:     !conf.GetBool("disableforceoverwrite"),
+		EnableMetrics:           conf.GetBool("enablemetrics"),
+		AnonymousGet:            conf.GetBool("authanonymousget"),
+		GenIndex:                conf.GetBool("genindex"),
+		MaxStorageObjects:       conf.GetInt("maxstorageobjects"),
+		IndexLimit:              conf.GetInt("indexlimit"),
+		Depth:                   conf.GetInt("depth"),
+		MaxUploadSize:           conf.GetInt("maxuploadsize"),
+		BearerAuth:              conf.GetBool("bearerauth"),
+		AuthRealm:               conf.GetString("authrealm"),
+		AuthService:             conf.GetString("authservice"),
+		AuthCertPath:            conf.GetString("authcertpath"),
+		AuthActionsSearchPath:   conf.GetString("authactionssearchpath"),
+		DepthDynamic:            conf.GetBool("depthdynamic"),
+		CORSAllowOrigin:         conf.GetString("cors.alloworigin"),
+		WriteTimeout:            conf.GetInt("writetimeout"),
+		ReadTimeout:             conf.GetInt("readtimeout"),
+		EnforceSemver2:          conf.GetBool("enforce-semver2"),
+		CacheInterval:           conf.GetDuration("cacheinterval"),
+		Host:                    conf.GetString("listen.host"),
+		PerChartLimit:           conf.GetInt("per-chart-limit"),
+		WebTemplatePath:         conf.GetString("web-template-path"),
+		ArtifactHubRepoID:       conf.GetStringMapString("artifact-hub-repo-id"),
+		KeepChartAlwaysUpToDate: conf.GetBool("keep-chart-always-up-to-date"),
```

Comment:
looks like we'll need this flag defined in `pkg/config/vars.go`? What do you think about calling it something like `--always-regen-index` or something similar 🤔.

Reply:
+1

Reply:
Okay I will rename it